FlowCrypt/Controllers/Search/SearchViewController.swift

```diff
@@ -17,26 +17,11 @@ import UIKit
  */
 final class SearchViewController: TableNodeViewController {
```

Comment:
Could you see if it's feasible / reasonable to completely remove `SearchViewController` and use `InboxViewController` for displaying search results? Alternatively, maybe subclass from `InboxViewController` ?

I'm asking in case it actually helps to deduplicate / simplify code - let me know what you think.

Reply:
Nice suggestion.
Yeah, I also thought of that and there are so many duplicated codes in SearchViewController.
Let me refactor